snuba/datasets/metrics_bucket_processor.py

```diff
@@ -76,14 +81,18 @@ def process_message(
         except EventTooOld:
             return None
 
+        processed_values = self._process_values(message)
```

Comment:
I guess another option would be to have a try catch here if you wanted to do the logic of the runtime config here. 

```python
try:
    process_values = self.__process_values(message)
except InvalidMessages as e:
    if state.get_config("enable_metrics_dlq", True):
        raise
    else:
      logger.error() # whatever you wanna log
      return None
```

But not sure if that's easier

Reply:
That is easier for `_process_values()` but I have the raise function being used a couple spots above too which would need a try/except for. I'm going to revert all this tomorrow after the prod test anyway so I don't think it's a big deal to clean this up for now.